spec/unit/shoulda/matchers/active_record/have_rich_text_matcher_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'unit_spec_helper'
+
+describe Shoulda::Matchers::ActiveRecord::HaveRichText, type: :model do
+  def self.rich_text_is_defined?
+    defined?(ActionText::RichText)
+  end
+
+  if rich_text_is_defined?
+    context 'when the model has a RichText association' do
+      it 'matches when the subject configures has_rich_text' do
+        valid_record = new_post(is_rich_text_association: true)
+
+        expected_message = 'Did not expect Post to have ActionText::RichText :content'
+
+        expect { have_rich_text(:content) }.
+          to match_against(valid_record).
+          or_fail_with(expected_message)
+      end
+
+      it 'returns the message including the name of the provided association' do
```

Comment:
Do you mind pulling this test out into its own `describe` block so that we know it has to do with `description` and not with running the matcher itself? I'm not sure it needs to be in this `if` block (it should be able to run regardless of whether `ActionText` is available or not). You could put the `describe` before the `if` statement on line 8.